src/index.ts

```diff
@@ -166,34 +135,72 @@ export class ErrorReporting {
      * // AFTER ALL OTHER ROUTE HANDLERS
      * server.register({register: errors.hapi});
      */
-    this.hapi = hapi(this._client, this._config);
+    this.hapi = hapiInterface.makeHapiPlugin(this._client, this._config);
```

Comment:
Why is it that `hapi` is a member field, but other plugins are not?

Reply:
The `hapi` property is an Object while the others are functions.